app/api/urls.py

```diff
@@ -52,6 +49,12 @@
          RoleMappingList.as_view(), name='rolemapping_list'),
     path('projects/<int:project_id>/roles/<int:rolemapping_id>',
          RoleMappingDetail.as_view(), name='rolemapping_detail'),
+    path('projects/<int:project_id>/docs/<int:doc_id>/comments',
+         CommentList.as_view(), name='comments_list'),
+    path('projects/<int:project_id>/comments',
+         CommentList.as_view(), name='comments_list_complete'),
+    path('projects/<int:project_id>/docs/<int:doc_id>/comments/<int:comment_id>',
+         CommentDetail.as_view(), name='comment_detail')
```

Comment:
move the path definitions back below the `AnnotationDetail` one. No need to move them around here.